libcodechecker/cmd/cmd_line_client.py

```diff
@@ -21,6 +21,8 @@
 from libcodechecker.logger import LoggerFactory
 from libcodechecker.output_formatters import twodim_to_str
 from libcodechecker.report import Report
+from libcodechecker import suppress_handler
```

Comment:
Import order is not proper here. `from libcodechecker import X` must precede `from libcodecher.a import b`.